WebDriverAgentLib/Utilities/FBConfiguration.h

```diff
@@ -46,6 +46,8 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)setShouldWaitForQuiescence:(BOOL)value;
 + (BOOL)shouldWaitForQuiescence;
 
++ (NSString* _Nullable)valueFromArguments: (NSArray<NSString *> *)arguments forKey: (NSString*)key;
```

Comment:
Please add docstring

Reply:
I would be happy to see descriptions for each argument and for the returned value. Otherwise looks good